guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -10,11 +10,13 @@ endif::[]
 . Click *Generate* to create the registration command.
 . Click on the _files_ icon to copy the command to your clipboard.
 . Connect to your host using SSH and run the registration command.
+. Ensure that the appropriate repositories have been enabled:
++
 ifdef::client-content-dnf[]
-. Check the `/etc/yum.repos.d/redhat.repo` file and ensure that the appropriate repositories have been enabled.
+** On {EL}: Check the `/etc/yum.repos.d/redhat.repo` file and ensure that the appropriate repositories have been enabled.
 endif::[]
 ifdef::client-content-apt[]
-. Check the `/etc/apt/sources.list` file and ensure that the appropriate repositories have been enabled.
+** On Debian: Check the `/etc/apt/sources.list` file and ensure that the appropriate repositories have been enabled.
 endif::[]
```

Comment:
The format does not seem that great if only one client content type is defined. Can there be separate ifdefs for deb, rpm, and katello instead?

Reply:
I've added more logic to accomodate this. But in downstream, I basically have the same issue: There is only _one_ content type active per render, be it dnf, apt, or zypper.

Maybe I can create a more elegant logic than this in the future!